hastory-cli/src/Hastory/Cli/Commands/GenGatherWrapper.hs

```diff
@@ -1,8 +1,53 @@
-module Hastory.Cli.Commands.GenGatherWrapper where
+module Hastory.Cli.Commands.GenGatherWrapper
+  ( genGatherWrapperScript
+  , genScript
+  ) where
+
+import Control.Monad (when)
+import Servant.Client.Core (BaseUrl(baseUrlScheme), Scheme(Http), parseBaseUrl)
 
 genGatherWrapperScript :: IO ()
-genGatherWrapperScript =
+genGatherWrapperScript = do
+  remoteInfo <- getRemoteInfo
+  putStrLn $ genScript remoteInfo
+
+getRemoteInfo :: IO (Maybe (String, String))
+getRemoteInfo = do
+  includeRemoteInfo <- promptForRemoteInfo
+  if includeRemoteInfo
+    then Just <$> gatherRemoteInfo
+    else pure Nothing
+
+promptForRemoteInfo :: IO Bool
+promptForRemoteInfo = do
   putStrLn $
+    unlines
+      [ "Would you like to include remote storage information (url and token)?"
+      , "Enter 1 for yes, 2 for no"
+      ]
+  userInput <- getLine
+  pure (userInput == "1")
+
+gatherRemoteInfo :: IO (String, String)
+gatherRemoteInfo = (,) <$> gatherBaseUrl <*> gatherToken
+
+gatherBaseUrl :: IO String
```

Comment:
As cool as this is, we don't want to make this an interactive this because it's not something that will be run by a user manually.

Reply:
Let's just make it so that no remote server is configured if the info is not available.
So no prompting.

Reply:
Will do.